src/js/Renderer.js

```diff
@@ -31,11 +31,12 @@ define([
       var className = options.className;
       var dropdown = options.dropdown;
       var hide = options.hide;
+      var btnStyle = options.btnStyle;
 
       return (dropdown ? '<div class="btn-group' +
                (className ? ' ' + className : '') + '">' : '') +
                '<button type="button"' +
-                 ' class="btn btn-default btn-sm btn-small' +
+                 ' class="btn ' + (btnStyle ? btnStyle : 'btn-default') + ' btn-sm btn-small' +
```

Comment:
If we provide a custom option for the class of buttons, someone may want to override whole style, not only `btn-default`, also sizes and others.

 - Option 1. Append custom class at the end of default classes.
  - e.g. `btn btn-default btn-sm btn-small`(default) + `btn-custom`(custom)
  - Custom classes will override previous default classes when they are in same category. (`btn-sm` / `btn-lg`)
 - Option 2. Or, replace whole classes with custom one.
  - e.g. `btn-my-class` (only, no default)